enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
@e-kolpakov Ok. Just curious that what pattern we are following for versioning scheme? http://semver.org